tdrs-backend/tdpservice/search_indexes/parsers/tanf_parser.py

```diff
@@ -0,0 +1,269 @@
+
+    DESCRIPTION         LENGTH  FROM    TO  COMMENTS
+    RECORD TYPE         2       1       2   "T1" - SECTION 1
+    REPORTING MONTH     6       3       8   Numeric
+    CASE NUMBER         11      9       19  Alphanumeric
+    COUNTY FIPS CODE    3       20      22  Numeric
+    STRATUM             2       23      24  Numeric
+    ZIP CODE            5       25      29  Alphanumeric
+    FUNDING STREAM      1       30      30  Numeric
+    DISPOSITION         1       31      31  Numeric
+    NEW APPLICANT       1       32      32  Numeric
+    FAMILY MEMBERS      2       33      34  Numeric
+    TYPE OF FAMILY      1       35      35  Numeric
+    SUBSIDIZED HOUSING  1       36      36  Numeric
+    MEDICAL ASSISTANCE  1       37      37  Numeric
+    FOOD STAMPS         1       38      38  Numeric
+    FOOD STAMP AMOUNT   4       39      42  Numeric
+    SUB CHILD CARE      1       43      43  Numeric
+    AMT CHILD CARE      4       44      47  Numeric
+    AMT CHIILD SUPPORT  4       48      51  Numeric
+    FAMILY'S CASH       4       52      55  Numeric
+    CASH
+    AMOUNT              4       56      59  Numeric
+    NBR_MONTH           3       60      62  Numeric
+    TANF CHILD CARE
+    AMOUNT              4       63      66  Numeric
+    CHILDREN_COVERED    2       67      68  Numeric
+    NBR_MONTHS          3       69      71  Numeric
+    TRANSPORTATION
+    AMOUNT              4       72      75  Numeric
+    NBR_MONTHS          3       76      78  Numeric
+    TRANSITIONAL SERVICES
+    AMOUNT              4       79      82  Numeric
+    NBR_MONTHS          3       83      85  Numeric
+    OTHER
+    AMOUNT              4       86      89  Numeric
+    NBR_MONTHS          3       90      92  Numeric
+    REASON FOR & AMOUNT OF ASSISTANCE
+    REDUCTION
+    SANCTIONS AMT       4       93      96  Numeric
+    WORK REQ            1       97      97  Alphanumeric
+    NO DIPLOMA          1       98      98  Alphanumeric
+    NOT IN SCHOOL       1       99      99  Alphanumeric
+    NOT CHILD SUPPORT   1       100     100 Alphanumeric
+    IRP FAILURE         1       101     101 Alphanumeric
+    OTHER SANCTION      1       102     102 Alphanumeric
+    PRIOR OVERPAYMENT   4       103     106 Alphanumeric
+    TOTAL REDUC AMOUNT  4       107     110 Alphanumeric
+    FAMILY CAP          1       111     111 Alphanumeric
+    LENGTH OF ASSIST    1       112     112 Alphanumeric
+    OTHER, NON-SANCTION 1       113     113 Alphanumeric
+    WAIVER_CONTROL_GRPS 1       114     114 Alphanumeric
+    TANF FAMILY
+    EXEMPT TIME_LIMITS  2       115     116 Numeric
+    CHILD ONLY FAMILY   1       117     117 Numeric
+    BLANK               39      118     156 Spaces
+    '''
+
+    # all of the below is assumed T1, we will eventually expand this for other sections
+    # via get_record_type(line)
+
+    # split line into fields
+    family_case_schema = RowSchema()
+    family_case_schema.add_fields(
```

Comment:
I agree with moe. The schemas are big enough that they warrant being imported from a separate directory.